bucket/impregnate.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.2.2",
+    "description": "A standalone installer for Cumcord.",
+    "homepage": "https://github.com/Cumcord/Impregnate",
+    "license": "Apache-2.0",
```

Comment:
License mentioned is MIT

https://github.com/Cumcord/Impregnate/blob/288f37d550218bdabc8dc4395c811a916d1170e9/LICENSE